openwhisk/openwhisk_apigateway.md

```diff
@@ -15,58 +15,111 @@ lastupdated: "2017-03-16"
 
-The OpenWhisk API Gateway integration allows you to proxy your Web Actions providing them with API Gateway features such as rate limiting, oauth token validation, API keys, and more. See [Create APIs from OpenWhisk actions](/docs/apis/management/manage_openwhisk_apis.html#manage_openwhisk_apis)
 
-Additionally, see the video [Create and Control APIs](https://youtu.be/XT9KwWTnnzo)
-
-**Note:** The APIs you created using the `wsk api-experimental` will continue to work, however you should begin migrating your APIs to web actions.
+**Note:** The APIs you created using the `wsk api-experimental` will continue to work for a short period, however you should begin migrating your APIs to web actions and reconfigure your existing apis using the new CLI command `wsk api`.
 
 ## OpenWhisk CLI configuration
-{: #openwhisk_apigateway_cli}
-This experimental feature only works with the new OpenWhisk authentication model in which each namespace now has a unique authentication key associated with it.
-Follow the instructions in [Configure CLI](https://console.ng.bluemix.net/openwhisk/cli) on how to set the authentication key for your specific namespace.
-
-## Expose an OpenWhisk action
-{: #openwhisk_apigateway_hello}
 
-Let's expose a simple action that is already pre-installed with OpenWhisk
-
-```
-wsk api-experimental create /hello /echo get /whisk.system/utils/echo
-```
-{: pre}
-```
-ok: created api /echo GET for action /whisk.system/utils/echo
-https://21ef035.api-gw.mybluemix.net/hello/echo
-```
-{: screen}
-A new URL is generated exposing the `echo` action via a __GET__ HTTP method.
-
-Let's give it a try by sending a HTTP request to the URL.
-```
-curl https://21ef035.api-gw.mybluemix.net/hello/echo?marco=polo
-```
-{: pre}
-This will invoke the `echo` action, returning back a JSON string with the parameters sent.
-```
-{
-  "marco":"polo"
-}
-```
-{: screen}
-
-You can pass parameters to the action via simple query parameters, or via request body.
-
-### Exposing multiple actions
-{: #openwhisk_apigateway_actions}
+Configure your OpenWhisk with the apihost `wsk property set --apihost openwhisk.ng.bluemix.net`
+To be able to use the `wsk api` the CLI configuration file `~/.wskprops` needs to contain the Bluemix Access Token.
+To get the access token use the CLI command `wsk bluemix login`, for more information about the command run `wsk bluemix login -h`
+
+Note: If the command errors requiring single sign on (sso), this is not currently supported. As a workaround login with the Bluemix CLI using `bluemix login`, then copy the Access Token from the HOME directory configuration file `~/.bluemix/config.json` into the `~/.wskprops` file as the property `APIGW_ACCESS_TOKEN="value of IAMToken`. Remove the prefix `Bearer ` when copying the access token string.
```

Comment:
`APIGW_ACCESS_TOKEN` value does not have surrounding quotes.

`APIGW_ACCESS_TOKEN=value-of-IAMToken-property`

If the org + space selected during `bluemix login` differs from the currently configured `wsk` namespace, the `wsk` CLI will need to be reconfigured (`https://console.ng.bluemix.net/openwhisk/learn/cli`)